test/hotspot/gtest/nmt/test_nmt_locationprinting.cpp

```diff
@@ -24,14 +24,14 @@
 
 #include "precompiled.hpp"
 #include "memory/allocation.hpp"
+#include "nmt/mallocHeader.inline.hpp"
+#include "nmt/memTracker.hpp"
 #include "runtime/os.hpp"
-#include "services/mallocHeader.inline.hpp"
-#include "services/memTracker.hpp"
+#include "testutils.hpp"
 #include "unittest.hpp"
 
 // Uncomment to get test output
 //#define LOG_PLEASE
-#include "testutils.hpp"
 
```

Comment:
There seems to be an ordering requirement between LOG_PLEASE and the inclusion of LOG_PLEASE. I think you break it with this change.